content/download/fedora.adoc

```diff
@@ -28,16 +38,5 @@ dnf install kicad
 If you don't have copr install with:
 
 ----
-yum install dnf-plugins-core
+dnf install dnf-plugins-core
 ----
-
-As a side note, the wxwidgets build in the copr build is forcing `--with=gtk2` on wxgtk, otherwise eeschema and the legacy canvas does not work properly. 
```

Comment:
I would still like to have the new copr builds verified. Also, do I need to make changes to https://github.com/kicad/fedora-packaging/blob/master/kicad.spec.template to make it use the proper wxwidgets version with gtk2?